src/components/grid/grid-item/grid-item.component.js

```diff
@@ -7,23 +7,29 @@ const GridItem = (props) => {
   const {
     children,
     responsiveSettings,
-    gridColumnStart,
-    gridColumnEnd,
-    gridRowStart,
-    gridRowEnd,
+    gridColumn,
+    gridRow,
     alignSelf,
-    justifySelf
+    justifySelf,
+    p,
+    pl,
+    pr,
+    pt,
+    pb
   } = props;
 
   return (
     <GridItemStyle
       { ...{
-        gridColumnStart,
-        gridColumnEnd,
-        gridRowStart,
-        gridRowEnd,
+        gridColumn,
+        gridRow,
         alignSelf,
-        justifySelf
+        justifySelf,
+        p,
```

Comment:
Same comment as on the container, are you specifically passing these down so we don't allow the margin props (and other padding props `px` and `py`) in this component?